src/saml20_clj/specs.clj

```diff
@@ -0,0 +1,14 @@
+(ns saml20-clj.specs
+  (:require [clojure.spec.alpha :as s])
+  (:import java.net.URL))
+
+(defn url? [s]
+  (try
+    (URL. s)
+    true
+    (catch Exception _
+      false)))
+
+
+(s/def ::acs-url url?)
+(s/def ::idp-url url?)
```

Comment:
does it make sense to have a specs namespace that has only some of the specs? I would prefer putting them all in a single namespace honestly. At Metabase for example we're still using Schema rather than spec, so we're not really going to use any of the spec stuff. It would be nice to have it out of the way for people who aren't planning on using it

Reply:
52e8acda342cbebf24c4b559e5c66fd3f76e76b3